common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala

```diff
@@ -191,11 +192,15 @@ object JavaHelpers {
   def createBoxCandidate(
                             value: Long, tree: ErgoTree,
                             tokens: util.List[ErgoToken],
-                            registers: util.List[Tuple2[String, Object]], creationHeight: Int): ErgoBoxCandidate = {
+                            registers: util.List[Object], creationHeight: Int): ErgoBoxCandidate = {
```

Comment:
@aslesarenko I could not figure out how to use `ErgoValue[T]` due to not being able to pull an allowed-in-java polymorphic type without a specifier in `OutBoxBuilderImpl._registers`, so I fallback to `Object`.

Reply:
fixed